### PR TITLE
feat: add webhook maxBytes setting

### PR DESCRIPTION
**Pull request by yuichi10**
## Context
Users try to start new PR event, but the build did not work because the webhook payload size was more than 1mb.
I make it possible to set maxByte size for webhook payload.

## Objective

- add `WEBHOOK_MAX_BYTES` in webhooks setting.

### CODE REVIEW

File: config/custom-environment-variables.yaml [hunk 1]
```diff
@@ -310,6 +310,8 @@ webhooks:
   restrictPR: RESTRICT_PR
   # Chain PR: true or false
   chainPR: CHAIN_PR
+  # Upper limit on incoming uploads to builds
+  maxByteSize: WEBHOOK_MAX_BYTES
```
Comment (sakka2) [inline config/custom-environment-variables.yaml]: maxByteSize か、maxBytes か、どちらかに名前を統一した方が良さそう。

File: plugins/webhooks/index.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ const ANNOT_RESTRICT_PR = `${ANNOT_NS}/restrictPR`;
 const CHECKOUT_URL_SCHEMA = require('screwdriver-data-schema').config.regex.CHECKOUT_URL;
 const CHECKOUT_URL_SCHEMA_REGEXP = new RegExp(CHECKOUT_URL_SCHEMA);
 const WAIT_FOR_CHANGEDFILES = 1.8;
+const DEFAULT_BYTES = 1048576;
```
Comment (sakka2) [inline plugins/webhooks/index.js]: [imo]`DEFAULT_MAX_BYTES` or `DEFAULT_MAX_BYTE_SIZE`の方が良さそう。

File: config/custom-environment-variables.yaml (hunk 1 above)
Comment (yuichi10) [reply config/custom-environment-variables.yaml]: I imitate store's setting 
https://github.com/screwdriver-cd/store/blob/master/config/custom-environment-variables.yaml#L31

Having same name of key and value is better. I will fix.

File: plugins/webhooks/index.js (hunk 2 above)
Comment (yuichi10) [reply plugins/webhooks/index.js]: I also imitate store's one.
https://github.com/screwdriver-cd/store/blob/master/plugins/builds.js#L11
I will fix.

File: config/custom-environment-variables.yaml (hunk 1 above)
Comment (yuichi10) [reply config/custom-environment-variables.yaml]: hapi use maxBytes. So I choose baxBytes.
https://github.com/hapijs/hapi/blob/ea5a4db1c773cf52643888dc347fa1c348254d5c/API.md#-routeoptionspayloadmaxbytes

File: plugins/webhooks/index.js (hunk 2 above)
Comment (yuichi10) [reply plugins/webhooks/index.js]: hapi use maxBytes. So I choose baxBytes.
https://github.com/hapijs/hapi/blob/ea5a4db1c773cf52643888dc347fa1c348254d5c/API.md#-routeoptionspayloadmaxbytes

File: config/custom-environment-variables.yaml (hunk 1 above)
Comment (sakka2) [reply config/custom-environment-variables.yaml]: OK!

File: plugins/webhooks/index.js (hunk 2 above)
Comment (sakka2) [reply plugins/webhooks/index.js]: OK!

**Pull request merged**